core/src/main/java/com/opensymphony/xwork2/ognl/OgnlUtil.java

```diff
@@ -57,17 +57,21 @@
     private final ConcurrentMap<Class, BeanInfo> beanInfoCache = new ConcurrentHashMap<>();
     private TypeConverter defaultConverter;
 
-    private boolean devMode = false;
+    private boolean devMode;                       // Defaults to false
+    private boolean devModeValueSet;               // Defaults to false (not explicitly set yet)
     private boolean enableExpressionCache = true;
-    private boolean enableEvalExpression;
+    private boolean enableEvalExpression;          // Defaults to false
+    private boolean enableEvalExpressionValueSet;  // Defaults to false (not explicitly set yet)
 
     private Set<Class<?>> excludedClasses;
     private Set<Pattern> excludedPackageNamePatterns;
     private Set<String> excludedPackageNames;
 
     private Container container;
-    private boolean allowStaticMethodAccess;
-    private boolean disallowProxyMemberAccess;
+    private boolean allowStaticMethodAccess;            // Defaults to false
```

Comment:
No need for the comments here.

Reply:
Don't be. :) Unnecessary comments just clutter code and can take focus away from the important stuff.